packs/lacework/config.yml

```diff
@@ -1,16 +1,16 @@
 id: 8a7a7220-e8ec-4959-b35d-0fe082be8039
 name: lacework
-description: End-to-end security management and diagnostics for major cloud platforms.
-summary: End-to-end security management and diagnostics for major cloud platforms.
+description: Integrate Lacework security data seamlessly into your observability experience.
```

Comment:
I think we might want to swap description and summary around. 
`summary` is used as a short text on the tile in the overview page of New Relic I/O
While `description` is intended as a longer form text that is shown once a user navigates to the details page of a quickstart. 
Does that make sense @lars2893 ?

Reply:
Note that markdown can be used in the description field to make it a bit fancier.

We also would like to see a short reference on how to get support for lacework in the description, maybe that's just calling out to go to support.lacework.com for support & more info?

Reply:
Could you beef up the description a bit more to align with the rest of our Quickstarts? Here's a recent partner quickstart with a great description: https://github.com/newrelic/newrelic-observability-packs/pull/207/files

You can find a template and style tips in the [contributing file](https://github.com/newrelic/newrelic-observability-packs/blob/main/CONTRIBUTING.md#quickstarts-style-guide). Let me know if you need any assistance or guidance!

Reply:
Helpful, feedback, I hadn't seen the UX yet, so wasn't entirely sure what to put there, I'll rev it per the documents passed along shortly!

Reply:
I noticed the referenced PR does not have an 'ID' guid, is this deprecated? I still have it from an earlier rev.